install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I don't think this is needed as `bash` is a superset of `sh`. Additionally, I'm not sure that our script is supported by bash (as it has some different syntax in some situations).

Reply:
like i said, [[ is a bashism. it is infact posix compliant and backwards compatible with posix scripts but when a bash script with bash features (such as [[) is run within a posix shell, it will error.

Reply:
which is why we use `sh`, to ensure maximum compatibility.

Reply:
then the script should be rewritten in POSIX in that case.

Reply:
opening new PR for a script rewritten in POSIX.